adapters/connectad/connectad.go

```diff
@@ -0,0 +1,213 @@
+package connectad
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type ConnectAdAdapter struct {
+	endpoint string
+}
+
+type connectadImpExt struct {
+	ConnectAd openrtb_ext.ExtImpConnectAd `json:"connectad"`
+}
+
+func (a *ConnectAdAdapter) SkipNoCookies() bool {
```

Comment:
`SkipNoCookies` is part of the legacy interface. It is no longer used. Please remove it.